src/index.html

```diff
@@ -4,14 +4,102 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
     <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+     method="post"
+     onsubmit="onSubmi()" required>
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <p>Surname:
+          <input type="text" id="surname" >
+          <label for="surname" class="field"></label>
+        </p>
+        <p>Name:
+          <input type="text" id="name" >
+          <label for="name" class="field"></label>
+        </p>
+        <p> How old are you?
+          <input type="number" id="number"
+           min="12" max="100"
+           value="12">
```

Comment:
```suggestion
          <input 
            type="number" 
            id="number"
            min="12"
            max="100"
            value="12"
          >
```
Форматируй элементы вот так, два отступа у атрибутов

Reply:
Форматувати у всьому коді,чи тільки тут?